server/redisstate.go

```diff
@@ -180,6 +188,39 @@ func (s *RedisState) GetSenderOfTxHash(txHash string) (txSender string, found bo
 	return strings.ToLower(txSender), true, nil
 }
 
+//
+// Enable lookup of tx bundles by bundle ID
+//
+func (s *RedisState) AddTxToWhitehatBundle(bundleId string, signedTx string) error {
+	key := RedisKeyWhitehatBundleTransactions(bundleId)
+
```

Comment:
small suggestion. declare variable as ctx := context.Background(); then It can be reused wherever required

Reply:
Not exactly sure what you mean - could you perhaps give an example?

Reply:
apologies for not being clear. simple change
`ctx := context.Background()
err := s.RedisClient.LPush(ctx, key, signedTx).Err()
	if err != nil {
		return err
	}

	// Set expiry
	err = s.RedisClient.Expire(ctx, key, RedisExpiryWhitehatBundleTransactions).Err()
	if err != nil {
		return err
	}

	// Limit to 15 entries
	err = s.RedisClient.LTrim(ctx, key, 0, 15).Err()
	return err
}`